yarn.lock

```diff
@@ -6959,7 +6959,7 @@ d3-zoom@1:
     d3-selection "1"
     d3-transition "1"
 
-d3@5.14.2:
+"d3@>=5.0.0 <=5.14.2":
```

Comment:
This is odd ... I think it stems from #1070 , but I would have expected it to be included there 🤔

Reply:
Yeah I was a bit perplexed by this; not sure why my `yarn install` causes this to change. I thought initially it might be because I was running an old version of yarn, but no, it still insists on making this change even after upgrading to `1.22.4`.

What do you think I should do here? Retain this change, or jimmy my `yarn.lock` file back to the original just before pushing?

Reply:
re `.storybook/__snapshots__/Welcome.story.storyshot`: I ran `yarn test:update` but still no update made to this file. Not sure what's going on here, does this need to be resolved before merging to master?

Reply:
Maybe try removing your `node_modules` directory and `yarn install` again to fix the d3 issue?

Reply:
I can repro the yarn.lock thing locally. Just leave it in - I mustve forgot to regen it in #1070.

Reply:
Ideally the welcome story will show all exports available as a frame of reference. I'd like to get it in there if we can - I'll pull down your branch and test it